common/customizable_localization/00_martial_lifestyle_custom_loc.txt

```diff
@@ -303,190 +303,190 @@ WarhorseStoryName = {
 		trigger = {
 			any_owned_story = {
```

Comment:
Initial event localization isn't fully done
![image](https://user-images.githubusercontent.com/74790101/234341411-9595befb-369d-47e7-b94b-97d4c54e191f.png)

First choice: Mentions "horse" and mentions some non-warcraft descriptions of names like Caligula
![image](https://user-images.githubusercontent.com/74790101/234341612-cd945ffa-213a-41f3-822e-6c9a5111f938.png)

Second choice: Some options go off the screen and are unclickable. 
![image](https://user-images.githubusercontent.com/74790101/234341911-a8dcefae-95c3-43aa-a478-3d32c6df253f.png)